.travis.yml

```diff
@@ -7,6 +7,8 @@ script:
         -ldflags "-X main.Version=$(git describe --tags)"
   - CGO_ENABLED=0 GOOS=linux GOARCH=arm GOARM=6 go build -o "mkcert-$(git describe --tags)-linux-arm"
         -ldflags "-X main.Version=$(git describe --tags)"
+  - CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -o "mkcert-$(git describe --tags)-linux-arm64"
+        -ldflags "-X main.Version=$(git describe --tags)"
```

Comment:
```suggestion
        -ldflags "-X main.Version=$(git describe --tags)"
  - CGO_ENABLED=0 GOOS=linux GOARCH=arm go build -o "mkcert-$(git describe --tags)-linux-arm"
        -ldflags "-X main.Version=$(git describe --tags)"
        
```

Reply:
@rfay but arm is already in there right? See above the `arm64`. Or is that the wrong version?

Reply:
Sorry, yes. I didn't even realize that arm was already there.